packages/Breadcrumbs/stories/Breadcrumbs.mdx

```diff
@@ -0,0 +1,119 @@
+import { Gap } from "storybook/assets/styles/common.styles";
+import Readme from "../README.md";
+import Breadcrumbs from "../src";
+
+<Description markdown={Readme} />
+
+### Basic Breadcrumbs
+
+```js
+<Breadcrumbs>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 1</Breadcrumbs.Link>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 2</Breadcrumbs.Link>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 3</Breadcrumbs.Link>
+</Breadcrumbs>
+```
+
+#### Result
+
+<Preview>
+  <Breadcrumbs>
+    <Breadcrumbs.Link>Breadcrumb 1</Breadcrumbs.Link>
+    <Breadcrumbs.Link>Breadcrumb 2</Breadcrumbs.Link>
+    <Breadcrumbs.Link>Breadcrumb 3</Breadcrumbs.Link>
+  </Breadcrumbs>
+</Preview>
+
+### Breadcrumbs will auto collapse if there're more than 3 items.
+
+```js
+<Breadcrumbs>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 1</Breadcrumbs.Link>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 2</Breadcrumbs.Link>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 3</Breadcrumbs.Link>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 4</Breadcrumbs.Link>
+</Breadcrumbs>
+```
+
+#### Result
+
+<Preview>
+  <Breadcrumbs>
+    <Breadcrumbs.Link>Breadcrumb 1</Breadcrumbs.Link>
+    <Breadcrumbs.Link>Breadcrumb 2</Breadcrumbs.Link>
+    <Breadcrumbs.Link>Breadcrumb 3</Breadcrumbs.Link>
+    <Breadcrumbs.Link>Breadcrumb 4</Breadcrumbs.Link>
+  </Breadcrumbs>
+</Preview>
+
+### Using isDark prop for breadcrumbs on a dark background
+
+```js
+<Breadcrumbs isDark>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 1</Breadcrumbs.Link>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 2</Breadcrumbs.Link>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 3</Breadcrumbs.Link>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 4</Breadcrumbs.Link>
+</Breadcrumbs>
+```
+
+#### Result
+
+<Preview>
+  <div style={{ padding: "16px", background: "#3f3d3c" }}>
+    <Breadcrumbs isDark>
+      <Breadcrumbs.Link>Breadcrumb 1</Breadcrumbs.Link>
+      <Breadcrumbs.Link>Breadcrumb 2</Breadcrumbs.Link>
+      <Breadcrumbs.Link>Breadcrumb 3</Breadcrumbs.Link>
+      <Breadcrumbs.Link>Breadcrumb 4</Breadcrumbs.Link>
+    </Breadcrumbs>
+  </div>
+</Preview>
+
+### Breadcrumbs will add a tooltip if the text in your Breadcrumb.Link longer than 40 characters
+
+```js
+<Breadcrumbs>
+  <Breadcrumbs.Link href={URL}>
+    Breadcrumb item with long text will have a tooltip. Hover this item to see.
+  </Breadcrumbs.Link>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 2</Breadcrumbs.Link>
+  <Breadcrumbs.Link href={URL}>Breadcrumb 3</Breadcrumbs.Link>
+</Breadcrumbs>
+```
+
+#### Result
+
+<Preview>
+  <Breadcrumbs>
+    <Breadcrumbs.Link>Breadcrumb item with long text will have a tooltip. Hover this item to see.</Breadcrumbs.Link>
+    <Breadcrumbs.Link>Breadcrumb 2</Breadcrumbs.Link>
+    <Breadcrumbs.Link>Breadcrumb 3</Breadcrumbs.Link>
+  </Breadcrumbs>
+</Preview>
+
+### Render Breadcrumbs.Link using your own components
+
+```js
+const MyButton = (children) => <button>{children}</button>
+
+<Breadcrumbs>
+  <Breadcrumbs.Link as="button" href={URL}>
```

Comment:
This is an interesting capability, but I can't think of a use case where we actually _should_ do this, for accessibility.  Breadcrumbs are navigational links, they should be rendered as `<a>` elements.  So I don't think we should advertise this, unless you can think of a legitimate use case?

Reply:
when the navigation bar is not jumping between pages and is a SPA without fully router support.

Reply:
Yes, I was thinking about the same scenario, devs might need to change the url in the onClick handler

Reply:
as well if you want to open a sidepanel from the breadcrumb instead of jumping into a new page. I think is useful not to attach it only to links

Reply:
I would argue it'd usually be a good idea for this state to have a dedicated URL. which would then be opened with a regular `<a>`.

Reply:
Seems weird to me that a breadcrumb would represent the state of the view with a sidepanel open.  I'm not 100% we would never do that, but I'd be sure to discuss with the UX designers about that pattern.

And I agree with what Alex said, even with an SPA, we should be using a unique URL and using an `<a>`.  Again, for accessibility, but also just for usability. That allows you to bookmark the view and use the browser navigation buttons.

But again, I'm not opposed to having this capability in case we need it, just don't think we should advertise it.

Reply:
Removed the example.